packages/shared/forks/Scheduler.umd.js

```diff
@@ -31,8 +31,11 @@ const {
   unstable_forceFrameRate,
 
   // this doesn't actually exist on the scheduler, but it *does*
-  // on scheduler/unstable_mock, which we'll need inside act().
+  // on scheduler/unstable_mock, which we'll need inside act()
+  // and for internal testing
   unstable_flushAllWithoutAsserting,
+  unstable_yieldValue,
+  _setDisableYieldValue,
```

Comment:
I don't think we want to export these in UMD builds either.

Reply:
Pending resolution to my comment on Scheduler, but if we add the functions there I think we will have to add them here too or they will fail tests. There's already some precedent on adding functions in `scheduler/unstable_mock` here though, so perhaps this is OK?

Reply:
Okay 👍🏼